test/node.js

```diff
@@ -11,8 +11,12 @@ import {foobarString} from './helpers/input.js';
 
 process.chdir(FIXTURES_DIR);
 
-const runWithNodeOption = (file, args = [], options = {}) => execa(file, args, {...options, node: true});
-const runWithNodeOptionSync = (file, args = [], options = {}) => execaSync(file, args, {...options, node: true});
+const runWithNodeOption = (file, args, options) => Array.isArray(args)
+	? execa(file, args, {...options, node: true})
+	: execa(file, {...options, node: true});
+const runWithNodeOptionSync = (file, args, options) => Array.isArray(args)
+	? execaSync(file, args, {...options, node: true})
+	: execaSync(file, {...options, node: true});
```

Comment:
This ensures we are testing both with and without `args`. 
There was a bug with not using `args`, and it did not get caught by the tests due to not trying both code paths.